### ISSUE TITLE
Alt text missing for 'Block I' SVG image

### ISSUE THREAD

**Issue by dandalpiaz** [web-illinois/ilw-header#13]
The image is being flagged for missing alt text. Apparently the inline `<svg>` element doesn't support the "alt" attribute.

Switched to use a `<title>` for the alt text with: https://github.com/web-illinois/ilw-header/pull/12
